src/twisted/names/test/test_names.py

```diff
@@ -1221,13 +1223,44 @@ def loadBindString(self, s):
         @return: a new bind authority
         @rtype: L{twisted.names.authority.BindAuthority}
         """
-        fp = FilePath(self.mktemp().encode("ascii"))
+        fp = FilePath(path)
```

Comment:
this should avoid updating existing test and should reduce the boilerplate code

```suggestion
        if no path:
            path = FilePath(self.mktemp()).asBytesMode().path
        fp = FilePath(path)
```

Reply:
Thanks for the suggestions and help. I appreciate.